cluster-http/src/main/scala/akka/management/cluster/ClusterHttpManagementRoutes.scala

```diff
@@ -143,4 +142,21 @@ object ClusterHttpManagementRoutes extends ClusterHttpManagementJsonProtocol {
       }
     )
 
+  /**
+   * Creates an instance of [[ClusterHttpManagementRoutes]] with only the read only routes.
+   */
+  def readOnly(cluster: Cluster): Route =
+    concat(
+      pathPrefix("cluster" / "members") {
+        concat(
+          pathEndOrSingleSlash {
+            routeGetMembers(cluster)
+          },
+          routesMember(cluster)
```

Comment:
perhaps rename this method to `routesGetMember` to make it more obvious that it's (currently) meant to be readonly?

Reply:
:+1: